### ISSUE TITLE
Notice on meta box generation

### ISSUE THREAD

**Issue by ghost** [TypeRocket/typerocket#6]
Hi, i found this Notice when WP_DEBUG is true:

```
Notice: Trying to get property of non-object in /home/divrenco/public_html/tobalai/wp-content/themes/tobalai/blackbox/frameworks/typerocket/src/MetaBox.php on line 124
```

So functions.php `require THEME_DIR . 'blackbox/wrapper.php';` and inside wrapper.php:

```
// TR_FOLDER is correct
require THEME_DIR . 'blackbox/frameworks/typerocket/init.php';

// a class to generate post type & metabox
require THEME_DIR . 'blackbox/wp/Typerocket.php';
```

the code of post type and metabox is also standard (this is wrapped inside a class):

```
/**
 * Register Post Type
 */
$produk = tr_post_type('Product', 'Products')
->setId('produk')
->setArgument('supports', array('title', 'editor'))
->setIcon('cart')
->setTitlePlaceholder('Enter the title here');

/**
 * Meta Box: Product Image
 */
tr_meta_box('Product Image')
->apply($produk)
->setCallback(function () {
    $form = tr_form();
    echo $form->image('product image')->setLabel('Upload image about 500 x 600 pixels');
});
```

When the meta box generation block is deleted, the notice was gone. May i know, how to solve that issue Kevin?

Thanks a lot,
Bagus

**Comment by kevindees** [comment]
Hey,

I have patched TypeRocket with a new version `2.0.11` addressing a few `WP_DEBUG` errors. Let me know if this update fixes your problems.

Thanks so much for reporting this error,
Kevin

**Comment by ghost** [comment]
Hi, patch `2.0.11` works perfectly: D
Thank you very much Kelvin, on the responses and solutions.

Thanks again,
Bagus
